backend/graphql/schema.graphql

```diff
@@ -65,3 +66,11 @@ Create a new user.
 type Mutation {
     createUser(user: CreateUserInput! @spread): User! @create
 }
+
```

Comment:
Is the branch up-to-date with development ?  Shouldn't createUser be here since it's since been merged into development ?

Reply:
Git reports this branch is up to date with development.